docs/conf.py

```diff
@@ -226,7 +226,11 @@
             "sphinx_jinja",
         ]
     )
-    exclude_patterns = ["operators/_partials"]
+    empty_subpackages = ["apache", "atlassian", "common", "cncf", "dbt", "microsoft"]
```

Comment:
Does Amazon need this treatment given the `aws` subdirectory it has?

Reply:
No. Because it does not have "amazon-aws" provider, only "amazon" provider. The others are separate providers, but if any code is added to "amazon.something_else" - they will still be in "amazon" provider. Unless we change the approach and have separate "amazon-aws" provider. This is what we did with "jira" provider that became "atlassian-jira" in the past. But for now, this is still single "apache-airflow-providers-amazon" provider and as such it cannot (and need not) be added here - mostly because anything "under" amazon is always in the same source tree - never in separate "amazon" packages in different providers.